source/backend/api/Areas/DocumentGeneration/DocumentGenerationController.cs

```diff
@@ -13,7 +14,7 @@ namespace Pims.Api.Controllers
     /// <summary>
     /// DocumentGenerationController class, provides endpoints to handle document generation requests.
     /// </summary>
-    [Authorize]
+    //[Authorize]
     [ApiController]
     [ApiVersion("1.0")]
     [Route("v{version:apiVersion}/documentGeneration/")]
```

Comment:
to follow our other format this may have been better as /document/generation/... instead

Reply:
I think is a sufficiently distinct functionality to warrant its own route. But not hard set on it.